tests/integration/tests.go

```diff
@@ -3088,6 +3089,23 @@ func PutObject_with_object_lock(s *S3Conf) error {
 	return nil
 }
 
+func PutObject_5gib_exceeding_file(s *S3Conf) error {
+	testName := "PutObject_5gib_exceeding_file"
+	return actionHandler(s, testName, func(s3client *s3.Client, bucket string) error {
+		obj := "my-obj"
+
+		_, err := putObjectWithData(5*1024*1024*1024+1, &s3.PutObjectInput{ // more than 5 Gib
```

Comment:
I think this is too large for github actions to handle. Maybe we need a separate test group for larger tests that we cant run automated in actions?